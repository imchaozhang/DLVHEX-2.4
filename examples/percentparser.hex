:- bar("foo","http://www.foo.com/bar%20baz").
:- foo("bar%").
foobar :- foo("%baz").
percent_parse_successful.
