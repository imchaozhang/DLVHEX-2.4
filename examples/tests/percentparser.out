{percent_parse_successful}
